packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { LIGHT_GRAY } from '../../utils/colors-values';
+import Text from '../../text';
+
+const horizontalPadding = '30px';
+
+export const Root = styled.div`
+	width: 272px;
+	padding: 35px 0px;
+	> *:not( .components-navigation__menu ) {
+		margin-left: ${ horizontalPadding };
+		margin-right: ${ horizontalPadding };
+	}
+`;
+
+export const Menu = styled.ul`
+	padding: 0;
+	margin: 0 0 32px 0;
+	display: flex;
+	flex-direction: column;
+`;
+
+export const MenuItemText = styled( Text )`
+	margin-right: auto;
+`;
+
+export const MenuItem = styled.li`
+	display: flex;
+	justify-content: space-between;
+	button {
+		width: 100%;
+		padding-top: 8px;
+		padding-bottom: 8px;
+		padding-left: ${ horizontalPadding };
+		padding-right: ${ horizontalPadding };
+	}
+	svg {
+		margin-left: 8px;
+	}
+	&.is-active ${ MenuItemText } {
+		border-bottom: 2px solid var( --wp-admin-theme-color );
+	}
+`;
+
+export const Badge = styled.span`
+	margin-left: 8px;
```

Comment:
Seems unnecessary

Reply:
This is needed to keep even spacing between elements if a menu item name is longer.  Note the spacing here is uneven:
<img width="232" alt="Screen Shot 2020-08-17 at 4 13 13 PM" src="https://user-images.githubusercontent.com/10561050/90440084-52d15200-e0df-11ea-846b-b972b8b99d42.png">

Keeping the margin on the badge is a better choice than the text so that elements are preemptively shortened if a badge does not exist.